src/cli-util.js

```diff
@@ -327,13 +327,20 @@ function formatFiles(argv) {
       // mtime based caches.
       if (output === input) {
         if (!argv["list-different"]) {
-          console.log(chalk.grey("%s %dms"), filename, Date.now() - start);
+          // For the tests, we only support the first argument for
+          // `console.log`. This means that the duration will appear as `%dms`
+          // in the snapshots, instead of for example `40ms`. This needs to be
+          // done since otherwise the tests would only pass (match the snapshot)
+          // if they happened to run in the exact amount of time as when
+          // created.
+          console.log(chalk.grey(`${filename} %dms`), Date.now() - start);
```

Comment:
Maybe we can mock `Date.now()` to always output `0` and replace `%d`/`%s` with their actual value?